pkg/security/keychain.go

```diff
@@ -156,12 +156,16 @@ func GetSecretFromKeyring(connectionID, uName string) (string, *SecError) {
 				return string(secret.Password), nil
 			}
 		}
-		err := errors.New("secret not found in keyring")
+		err := errors.New(textSecretNotFound)
 		return "", &SecError{errOpKeyring, err, err.Error()}
 	}
 	// else get from system keyring
 	secret, err := keyring.Get(service, uName)
 	if err != nil {
+		if err == keyring.ErrNotFound {
+			err := errors.New(textSecretNotFound)
+			return "", &SecError{errOpKeyring, err, err.Error()}
```

Comment:
I'm assuming in go this `err` refers to the one defined on the line above, not the one defined 3 lines above